plugins/out_s3/s3.c

```diff
@@ -1113,6 +1240,19 @@ static int s3_put_object(struct flb_s3 *ctx, const char *tag, time_t create_time
             return -1;
         }
     }
+
+    /* Update file and increment index value right before request */
+    if (ctx->key_fmt_has_seq_index) {
```

Comment:
this feature works for both PutObject and for multipart uploads? Or no?

I think i see that this code has only been added for the PutObject case...

Reply:
@StephenLeeY ?

Reply:
Multipart has been added!